scipy/optimize/_differentialevolution.py

```diff
@@ -1244,16 +1255,20 @@ def solve(self):
                                   "infeasible solution",
                                   UserWarning, stacklevel=2)
             if self.disp:
-                print(f"Polishing solution with '{polish_method}'")
+                print(f"Polishing solution with '{self.minimizer_kwargs['method']}'")
             result = minimize(lambda x:
                                 list(self._mapwrapper(self.func, np.atleast_2d(x)))[0],
                               np.copy(DE_result.x),
-                              method=polish_method,
                               bounds=self.limits.T,
-                              constraints=self.constraints)
+                              constraints=self.constraints,
+                              **self.minimizer_kwargs)
```

Comment:
If there are duplicate keys in `minimizer_kwargs` then this call will fail. i.e. if the dict contains `bounds`/`constraints`, `fun`, `x0`, etc. Perhaps it's better to populate the dict will *all* the parameters to be used with `minimize`.

Reply:
You'd be surprised at the variety of usage modes out there. Sometimes people use dicts to carry all the parameters to be given to a function, which would include x0, fun.

Reply:
Okay, I made more changes: now everything is put into the `self.minimizer_kwargs` dict, including `fun` and `x0`. If a user put `fun` or `x0` into `minimizer_kwargs`, they are overridden and a warning is displayed.